reverse.js

```diff
@@ -0,0 +1,12 @@
+let arr = [1, 2, 3, 6, 4];
+const num = arr.length;
```

Comment:
because you used the length of particular array, your function will work only for this array.